hw1/01-fizzbuzz.js

```diff
@@ -1,8 +1,17 @@
+//Description : Program to print Fizz if a number is divisible by 3, Buzz if a number
+//is divisible by 5 and Fizz Buzz if it is divided by 3 and 5 both
+
 const fizzbuzz = () => {
-  // Add your code here
-};
+    for (let number = 1; number <= 100; number += 1) {
+        let wordToPrint = number
+        if (number % 3 === 0 && number % 5 === 0) wordToPrint = 'Fizz Buzz'
+        else if (number % 3 === 0) wordToPrint = 'Fizz'
+        else if (number % 5 === 0) wordToPrint = 'Buzz'
+        console.log(wordToPrint)
```

Comment:
Small change. printed words are capitalized and there is space between fizzbuzz. 
Nothing major. It is just that, had this been setup with test units and all, program would have failed. 

Rest looks good to me.

Reply:
Updated. I didn't see that as the major requirement, that's why wrote it like that.

Reply:
Approved.